components/badge/badge.vue

```diff
@@ -97,12 +102,25 @@ export default {
       default: 'default',
       validator: (type) => Object.keys(BADGE_TYPE_MODIFIERS).includes(type),
     },
+
+    /**
+     * Decoration for the badge
```

Comment:
Just a suggestion, is it better to mention here the decoration prop can only be used with kind: `label` and type `default` and no `iconLeft` and `iconRight` so eng can easily know such limitations? 

![image](https://user-images.githubusercontent.com/94570795/225662898-0793855e-d972-422d-9412-9b17f03a78ea.png)

Reply:
Yes for sure.